igneous/tasks.py

```diff
@@ -266,14 +266,21 @@ def __init__(self, shape, offset, layer_path, **kwargs):
         'high_padding': kwargs.get('high_padding', 1),
         'parallel_download': kwargs.get('parallel_download', 1),
         'cache_control': kwargs.get('cache_control', None),
-        'use_draco_encoding': kwargs.get('use_draco_encoding', False),
+        'encoding_type': kwargs.get('encoding_type', 'precomputed'),
     }
     self.draco_encoding_settings = {
       'quantization_bits': kwargs.get('draco_quantization_bits', 14),
       'compression_level': kwargs.get('draco_compression_level', 1),
       'quantization_range': kwargs.get('draco_quantization_range', -1),
       'quantization_origin': kwargs.get('draco_quantization_origin', None)
     }
+    supported_encodings = ['precomputed', 'draco']
+    if not self.options['encoding_type'] in supported_encodings:
+      raise ValueError('Encoding type specified is not supported')
```

Comment:
The encoding type should be shown in the error message, otherwise you might drive yourself crazy debugging. :sob: